service-config/src/main/java/com/palantir/config/service/CustomImmutableStyle.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+ */
+
+package com.palantir.config.service;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.immutables.value.Value.Style;
+
+/**
+ * Custom style for immutable classes.
+ */
+@Target({ElementType.PACKAGE, ElementType.TYPE})
+@Retention(RetentionPolicy.CLASS)
+@Style(
+    visibility = Style.ImplementationVisibility.PACKAGE)
+@interface CustomImmutableStyle {}
```

Comment:
Name is pretty generic -- would prefer it either communicate the functionality (`PackageAccessImmutable`, `PackageVisibilityImmutableStyle`, etc.) or project (`HttpRemotingImmutable`)